bindings/matrix-sdk-ffi/src/room.rs

```diff
@@ -936,6 +922,29 @@ impl Room {
         });
     }
 
+    pub fn get_event_timeline_item_by_event_id(
+        &self,
+        event_id: String,
+    ) -> Result<Arc<EventTimelineItem>, ClientError> {
+        RUNTIME.block_on(async move {
```

Comment:
Why blocking? Wouldn't be fine to make it async?

Reply:
I don't want to introduce more async functions until the memory leak problems in Kotlin are fixed, and the only thing this `.await`s is a lock on timeline internals.

Reply:
Then fine.